src/website/shared/models.py

```diff
@@ -255,14 +257,29 @@ class Type(models.TextChoices):
 
 ###
 #
-# Nix related models
+# Nixpkgs related models
 #
 ##
 
 
-class NixIssue(models.Model):
+class NixpkgsIssue(models.Model):
     """The Nixpkgs version of a cve."""
 
+    @property
+    def issue_id(self):
+        return f"NIXPKGS-{str(self.created.year)}-{str(self.number).zfill(5)}"
```

Comment:
todo: the short code is a settings of the deployment.

Reply:
Broke out an issue